source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
```

Comment:
I don't think POST is appropriate until we move export to the progress endpoint, regardless of URL size (although I do agree the size of a URL is a limitation)